notebooks/visit_data.py

```diff
@@ -0,0 +1,255 @@
+# can we predict the next site to be visited from the current site?
+# this is a hard high cardinality challenge, so at random, the predictive performance is low
+# is this flawed anyway, as the results being presented just reflect what the user was limited to selecting
+# like predicting that the bus is red
+# have a performance measurement at random, and using the model results
+# are the parent pagePaths sufficiently granular
+
+import pandas as pd
+import os
+import numpy as np
+import re
+from flatten_dict import unflatten, flatten
+
+os.chdir("/home/james/Downloads")
+data_dir = '/home/james/Documents/gds_nlp/govuk-entity-personalisation/data'
```

Comment:
To generalise this, we can tweak to the following below as relative paths to the project should be picked up:
```suggestion
data_dir = 'data'
```